elastalert/elastalert.py

```diff
@@ -1105,7 +1105,7 @@ def get_aggregated_matches(self, _id):
         """ Removes and returns all matches from writeback_es that have aggregate_id == _id """
 
         # XXX if there are more than self.max_aggregation matches, you have big alerts and we will leave entries in ES.
-        query = {'query': {'query_string': {'query': 'aggregate_id:%s' % (_id)}}}
+        query = {'query': {'query_string': {'query': 'aggregate_id:%s' % (_id)}}, "sort": {"@timestamp": "asc"}}
```

Comment:
The other dictionary fields have single quotes. You should have these match for consistency sake